src/titiler_fetcher.mjs

```
@@ -3,7 +3,7 @@ import PQueue from "p-queue";
 import os from "node:os";
 
 // Getting the number of cores
-const numCPUs = os.cpus().length;
+const numCPUs = process.env.NUM_CPUS ? parseInt(process.env.NUM_CPUS, 10) : os.cpus().length;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)